pyproject.toml

```diff
@@ -19,7 +19,9 @@ classifiers = [
     "Operating System :: OS Independent",
 ]
 dependencies = [ 
-  "tqdm"
+  "tqdm",
+  "pyparsing",
```

Comment:
pyparsing is needed for parsing Petri nets with data and does not have much to do with the conformance checking implementation in this issue. But is required to do the conformance checking. May just be overthinking dependencies.

Reply:
It's worth being conservative with dependencies in a utility library that isn't an end-project because they will propagate everywhere. It's not worth worrying about so much in a client / leaf project unlikely to be imported by anything else.

Reply:
Sure, but we going from one to three? Joblib has board use cases as the best in the slot for parallelism in Python, and pyparsing allows for grammars, which will be handy in the future for many things.